rust/lance-linalg/src/kmeans.rs

```diff
@@ -432,6 +398,7 @@ impl KMeans {
     ///   let kmeans = membership.to_kmeans();
     /// }
     /// ```
+    #[instrument(level = "debug", skip(self))]
```

Comment:
```suggestion
    #[instrument(level = "debug", skip(self, data))]
```
Or you can do...
```
    #[instrument(level = "debug", skip_all)]
```

Otherwise it will log the debug representation of the matrix view which might be large.

Reply:
Kk, i will do `skip_all` then

Reply:
Fixed.